core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1314,9 +1314,9 @@ Traders can communicate within the application via secure chat on the open trade
 support.initialInfo=Please enter a description of your problem in the text field below. \
   Add as much information as possible to speed up dispute resolution time.\n\n\
   Here is a check list for information you should provide:\n\
-  \t● If you are the BTC buyer: Did you make the Fiat or Altcoin transfer? If so, did you click the 'payment started' \
+  \t● If you are the BTC buyer: Did you make the Fiat or Altcoin transfer? If so, did you click the ''payment started'' \
```

Comment:
Here is no variable used so the single quote is correct. Otherwise it would be displayed as double quote.

Reply:
It's a big, multi-line string.  The variable is 10 lines down on line 1327 in this case.  All these apostrophe fixes were identified using a script to analyze the properties files.

Reply:
For the string `support.initialInfo` you specifically questioned here:

**BEFORE:**
![Screenshot from 2024-10-11 23-30-26(1)](https://github.com/user-attachments/assets/9a892ffa-f899-46d4-a051-d7e1c149de10)

**AFTER:**
![Screenshot from 2024-10-12 01-20-25](https://github.com/user-attachments/assets/0c1893e2-4d43-41a4-b7bf-0d56465236ff)

This was an unusual one because it originally had both single apostrophe's and double apostrophe's in the same string! (and I had not changed anything about it in my original commit on this branch)

I think we're good.